src/graph.js

```diff
@@ -3,6 +3,38 @@ class Graph {
   constructor() {
     this.graph = {};
   }
+
+addNode(value) {
```

Comment:
Please try to avoid wonky indentation. All of these methods need to be indented once over so that people reading your code can easily tell that they are all methods on this class.